src/App.js

```diff
@@ -1,12 +1,24 @@
 import { PureComponent, Fragment } from 'react';
+import { Route, Switch } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Navbar from './components/NavBar';
+import Home from './pages/Home';
+import Quote from './pages/Quote';
 import './App.css';
 
 class App extends PureComponent {
   render() {
     return (
       <>
-        <Calculator />
+        <Navbar />
+        <Switch>
+          <Route
+            path="/calculator"
+            component={Calculator}
+          />
+          <Route exact path="/" component={Home} />
+          <Route path="/qoute" component={Quote} />
+        </Switch>
       </>
     );
   }
```

Comment:
- [ ] KIndly make sure at when the application starts the Home component should be rendered as the root component. Actually, when your application starts I am getting only the navigation bar :point_down:

![image](https://user-images.githubusercontent.com/65068771/154769139-574b6285-5477-4fad-be6f-035e7cdea240.png)

- [ ] You can solve it by removing the exact key world on the Home route.